### PR TITLE
[zh] sync certificate-signing-requests feature-gates cheatsheet

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -1252,17 +1252,19 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
 -->
 - `SELinuxMountReadWriteOncePod`：通过允许 kubelet 直接用正确的 SELinux
   标签为 Pod 挂载卷而不是以递归方式更改这些卷上的每个文件来加速容器启动。最初的实现侧重 ReadWriteOncePod 卷。
-- `SchedulerQueueingHints`：启用调度器的**排队提示**增强功能，有助于减少无效的重新排队。
-- `SeccompDefault`: 允许将所有工作负载的默认  seccomp 配置文件为 `RuntimeDefault`。
-  seccomp 配置在 Pod 或者容器的 `securityContext` 字段中指定。
+- `SchedulerQueueingHints`：启用[调度器的排队提示增强功能](https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/4247-queueinghint/README.md)，
+  有助于减少无效的重新排队。调度器会在集群中发生可能导致 Pod 被重新调度的变化时，
+  尝试重新进行 Pod 的调度。排队提示是内部信号，用于帮助调度器筛选与未调度的 Pod 相关的集群变化，
+  根据先前的调度尝试来判断其是否相关。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:1266]:
```suggestion
  尝试重新进行 Pod 的调度。排队提示是一些内部信号，
  用于帮助调度器基于先前的调度尝试来筛选集群中与未调度的 Pod 相关的变化。
```

File: content/zh-cn/docs/reference/kubectl/cheatsheet.md [hunk 2]
```diff
@@ -476,6 +479,9 @@ kubectl get pods --show-labels
 # 检查哪些节点处于就绪状态
 JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}' \
  && kubectl get nodes -o jsonpath="$JSONPATH" | grep "Ready=True"
+ 
+# 检查哪些节点处于就绪状态，使用自定义列
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubectl/cheatsheet.md:483]:
```suggestion
# 使用自定义列检查哪些节点处于就绪状态
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**